incubator/virtualcluster/config/sampleswithspec/clusterversion_v1.yaml

```diff
@@ -9,64 +9,67 @@ spec:
   etcd:
     metadata:
       name: etcd 
-      namespace: vc-test
     statefulset:
       metadata:
-        name: vc-test/statefulset/etcd
+        name: etcd
       spec:
-        replicas: 1
+        replicas: 3
```

Comment:
This will definitely cause you issues in production. If you have an issue with the first or second replica you have no way to remove/re-add a corrupted member easily. But as this is a sample I'm sure it's ok. Just something I've ran into in the past!

Reply:
Won't statefulset controller restart fail pod?

Reply:
Did you mean it will be difficult to control each individual pod (eg. reschedule failed pod to another node) of the statefulset?